Sources/X509/Digests.swift

```diff
@@ -43,6 +43,22 @@ enum Digest {
     }
 }
 
+extension Digest: Sequence {
+    @usableFromInline
+    func makeIterator() -> Array<UInt8>.Iterator {
```

Comment:
It might be nicer to return this using `some IteratorProtocol`, potentially with the extra `associatedtype` to get that to work. That adds some defense in-case we accidentally make this public later and we regret exposing the array iterator directly.

Reply:
Good idea.